scripts/release.mjs

```diff
@@ -29,13 +29,15 @@ import { execa } from 'execa';
   }
 
   const { workspaceVersion, projectsVersionData } = await releaseVersion({
+    projects: ['tag:type:package'],
```

Comment:
@SokratisVidros is this script intended to release packages only? If not, we need to remove this filter

Reply:
It releases only packages.